robots/ComauSmartSixRobot.m

```diff
@@ -32,13 +32,15 @@
             comau_DH_d =     [-0.45, 0, 0, -0.64707, 0, -0.095];
             comau_DH_a =     [0, 0.150, 0.590, 0.13, 0, 0];
             comau_DH_alpha = [pi, pi/2, pi, -pi/2, -pi/2, pi/2];
+            comau_DH_type =  [1,1,1,1,1,1];
 
             comau_DH_matrix = [comau_DH_theta;
                 comau_DH_d;
                 comau_DH_a;
-                comau_DH_alpha];
+                comau_DH_alpha
+                comau_DH_type];
 
-            comau = DQ_SerialManipulator(comau_DH_matrix, 'modified');
```

Comment:
It seems that those parameters are the modified ones (I double-checked the current code). Therefore, if we instantiate this robot using these parameters with the standard DH convention, we'll get the wrong results for the forward kinematics.

Reply:
TODO 

- [ ] Update ComauSmartSixRobot to use `DQ_JointType`

- [ ] Update ComauSmartSixRobot to use the `DQ_SerialManipulatorMDH()`.

- [ ] Add the class `DQ_SerialManipulatorMDH()`.